src/components/Tabs/tabs.module.css

```diff
@@ -0,0 +1,52 @@
+.tabsContainer {
+    composes: pl-8 from global;
+    composes: pr-8 from global;
+}
+.tabs {
+    composes: flex from global;
+    composes: flex-wrap from global;
+    composes: border-b from global;
+    composes: border-gray-grayCoose from global;
+}
+
+.tab {
+    composes: font-bold from global;
+    composes: pt-4 from global;
+    composes: pb-4 from global;
+    composes: pl-8 from global;
+    composes: pr-8 from global;
+    composes: cursor-pointer from global;
+    border: 1px solid #DFE1E2;
+    border-bottom: 0px;
+}
+
+.activeTab {
+    composes: bg-gray-tabActive from global;
+}
+
+.textContainer {
+    composes: pt-6 from global;
+    composes: pb-4 from global;
+}
+
+.title {
+    composes: font-bold from global;
+}
+
+.descriptionText {
+    composes: mt-3 from global;
+    composes: text-sm from global;
+}
+
+@media (min-width: 1024px) {
+
+    .tabsContainer {
+       padding: 0px;
+    }
+
+    .tab {
+        composes: pl-10 from global;
+        composes: pr-10 from global;
+    }   
```

Comment:
Should use breakpoints from tailwind instead of @media https://tailwindcss.com/docs/responsive-design basically `md_pl-10` or so, depending what kind of min-width you need.